packages/dds/tree/src/test/utils.ts

```diff
@@ -1300,3 +1302,23 @@ export function moveWithin(
 ) {
 	editor.move(field, sourceIndex, count, field, destIndex);
 }
+
+/**
+ * Clones a batch of revertibles for a target branch.
+ * @param revertibles - Array of revertibles to clone
+ * @param targetBranch - The target branch to clone the revertibles for
+ * @returns Array of cloned revertibles, maintaining the same order as the input
+ */
+export function cloneRevertiblesInBatch(
```

Comment:
This needs to be package exported for users to be able to use it

Reply:
(with the alpha tag)

Reply:
Made changes while you were reviewing 😄. Would you look into this again please?